### PR TITLE
refactor: Add deprecation notice for MySQL and MariaDB

### PR DESCRIPTION
**Pull request by krynble**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/cli/src/commands/BaseCommand.ts [hunk 1]
```diff
@@ -55,6 +55,14 @@ export abstract class BaseCommand extends Command {
 			this.exitWithCrash('There was an error initializing DB', error),
 		);
 
+		const dbType = config.getEnv('database.type');
+
+		if (['mysqldb', 'mariadb'].includes(dbType)) {
+			LoggerProxy.warn(
+				'MySQL/MariaDB has been deprecated and will have support removed when n8n 1.0 is released. Please migrate to PostgreSQL.',
```
Comment (csuermann) [inline packages/cli/src/commands/BaseCommand.ts:62]:
```suggestion
				'Support for MySQL/MariaDB has been deprecated and will be removed with an upcoming version of n8n. Please migrate to PostgreSQL.',
```

**Review by csuermann — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.228.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.228.2)

**Comment by luizeof** [comment]
Hello @jan, is there any prediction of version or date for mysql to become deprecated? This is important to organize the community to migrate to postgres.

Will SQLite be maintained?

**Comment by Joffcom** [comment]
Hey @luizeof 

It is being removed in 1.0 which will be available in a few weeks.

SQLite will remain as the default option. You can find a list of the breaking 1.0 changes here: https://docs.n8n.io/1-0-preview/

**Comment by csuermann** [comment]
Quick update: We decided **not** to remove support for MySQL / MariaDB in the upcoming version 1 just yet, but it is still advisable to switch to PostgreSQL or SQLite.

**Comment by Soetens** [comment]
> Quick update: We decided **not** to remove support for MySQL / MariaDB in the upcoming version 1 just yet, but it is still advisable to switch to PostgreSQL or SQLite.

To be sure, this is about the storage of n8n itself and not about the mysql node?

**Comment by Joffcom** [comment]
@Soetens you got it 👍 this is just for the backend and not the node.

**Pull request merged**